android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -210,9 +202,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         setUpToolbar();
         setUpNavDrawer();
-        setUpProgressBar();
         setUpCustomTab();
 
+        completeHandler = DownloadCompleteHandler.with(context);
```

Comment:
See Picasso and other modern libraries. When not using this, it may confuse the user. How to create the object, which method to call first?

This provides a natural order with chaining configuration